### ISSUE TITLE
Make.md conflict with Folder Icons plugin causes icons to disappear

### ISSUE THREAD

**Issue by DRD161** [Make-md/makemd#118]
I had an issue where folder icons were disappearing when editing notes or closing and reopening Obsidian. Initially i suspected the Folder Icons plugin itself but after taking a look at the Obsidian dev console I found that I was receiving an error related to the `handleChangeLayout()` function for the Folder Icons plugin. I thought about what plugins I had installed that changed the layout of Obsidian and since Make.md adds Spaces and other layout changes I figured that'd be the best place to start.

After disabling the Make.md plugin I found that the error went away and folder icons persisted as they should. Both are fantastic plugins and it'd be great if they could work together. I will link to the issue I filed for the Folder Icons plugin and hopefully this can be resolved soon.

**Console Error**

<img width="549" alt="Screenshot 2023-05-05 at 9 09 52 AM" src="https://user-images.githubusercontent.com/29187448/236472678-5e88b57c-7050-48fc-885f-76c1cd10863b.png">

**Link to Folder Icons Issues**

https://github.com/FlorianWoelki/obsidian-icon-folder/issues/174

**Comment by jp-cen** [comment]
The Spaces Compatibility Setting solves this issue

**Comment by DRD161** [comment]
@jp-cen I tried that and unfortunately it does not solve the issue.

**Comment by jp-cen** [comment]
Have you tried restarting Obsidian after toggling the setting? The settings currently only take effect after restart

**Comment by DRD161** [comment]
Yes I tried that as well. I found that after adding an icon and enabling Compatibility mode that the icons will persist but only when viewing notes by files, the icons still disappear when viewing by Spaces.

**Files view - Icons shown**
<img width="270" alt="Screenshot 2023-05-05 at 9 58 07 AM" src="https://user-images.githubusercontent.com/29187448/236478498-b865443d-5345-4516-b574-cfb3836e5e79.png">

**Spaces view - Icons missing**
<img width="159" alt="Screenshot 2023-05-05 at 10 00 19 AM" src="https://user-images.githubusercontent.com/29187448/236479015-95c9f0ae-58d5-4baf-918f-4eeab6b65064.png">

**Comment by DRD161** [comment]
Ahh got it. I appreciate the info, looking forward to the custom icons feature!
